apps/legacy/src/pages/Swap/components/TransactionDetails.tsx

```diff
@@ -150,79 +162,91 @@ export function TransactionDetails({
+                >
+                  <Typography
+                    variant="body2"
+                    sx={{
+                      my: 0,
+                      mr: 1,
+                      ml: 0,
+                      fontWeight: theme.typography.fontWeightSemibold,
+                    }}
+                  >
+                    {t('Slippage tolerance')}
+                  </Typography>
+                  <SlippageToolTip />
+                </Stack>
+                <Stack
+                  sx={{ flexDirection: 'column', gap: 0.5, width: '100%' }}
+                >
+                  <TextField
+                    data-testid="swap-slippage-tolerance-input"
+                    size={'small'}
+                    value={slippage}
+                    placeholder="Input Percent %"
+                    fullWidth
+                    type="number"
+                    InputProps={{
+                      inputProps: {
+                        min: MIN_SLIPPAGE,
+                        max: 100,
+                        step: 0.1,
+                      },
+                    }}
+                    onChange={(e) => {
+                      const inputValue = e.target.value;
+                      setSlippage(inputValue);
 
-                  if (isSlippageValid(inputValue)) {
-                    setError('');
-                  } else {
-                    setError(t('Enter a value of at least 0.1%'));
-                  }
-                }}
-              />
-              <Typography variant="caption" color="error.main" minHeight="14px">
-                {error}
-              </Typography>
-            </Stack>
-          </Stack>
-          {swapFeesInfo.isCollectingFees && (
-            <DetailsRow
-              sx={{
-                mt: 2,
-                justifyContent: 'start',
-              }}
-            >
-              <Typography variant="caption" color="text.secondary">
-                {t('Quote includes a {{formattedFeePercent}} Core fee', {
-                  formattedFeePercent: swapFeesInfo.percentage,
-                })}
-              </Typography>
-              <Tooltip
-                sx={{
-                  px: 1,
-                }}
-                isInfo
-                title={t(
-                  'Core always finds the best price from the top liquidity providers. A fee of {{formattedFeePercent}} is automatically factored into this quote.',
-                  {
-                    formattedFeePercent: swapFeesInfo.percentage,
-                  },
-                )}
-              >
-                <InfoCircleIcon sx={{ color: 'text.secondary' }} />
-              </Tooltip>
-            </DetailsRow>
+                      if (isSlippageValid(inputValue)) {
+                        setError('');
+                      } else {
+                        setError(t('Enter a value of at least 0.1%'));
+                      }
```

Comment:
What about ternary expression here?
```suggestion
                    setError(isSlippageValid(inputValue) ? '' :  t('Enter a value of at least 0.1%'));

```